GitDepend/Resources/strings.resx

```diff
@@ -123,6 +123,12 @@
   <data name="RET_CREATE_CACHE_DIR_FAILED" xml:space="preserve">
     <value>Failed to create the nuget cache directory</value>
   </data>
+  <data name="RET_DEPENDENCY_PACKAGES_MISTMATCH" xml:space="preserve">
```

Comment:
These should only be necessary if this case is a fatal error that will stop program execution and give the user an error.